pkg/bbgo/interact.go

```diff
@@ -410,12 +411,12 @@ func getStrategySignature(strategy SingleExchangeStrategy) (string, error) {
 		return "", fmt.Errorf("strategy %T instance is not a struct", strategy)
 	}
 
-	var signature = path.Base(rv.Type().PkgPath())
-
-	var id = strategy.ID()
-
-	if !strings.EqualFold(id, signature) {
-		signature += "." + strings.ToLower(id)
+	var signature string
+	var id = callID(strategy)
+	if id != "" {
+		signature = id
```

Comment:
you can return directly? since the instance ID should already be unique identifier

Reply:
So we don't need interval in the signature?

Reply:
if we have instance id, we don't need to build the signature.